mavros_extras/src/plugins/vision_speed_estimate.cpp

```diff
@@ -76,38 +78,53 @@ class VisionSpeedEstimatePlugin : public plugin::PluginBase {
 		vs.y = v.y();
 		vs.z = v.z();
 		// [[[end]]] (checksum: aee3cc9a73a2e736b7bc6c83ea93abdb)
+		ftf::covariance_to_mavlink(cov, vs.covariance);
```

Comment:
Leave blank lines before and after cog block.

Reply:
Any good reason for it or is just for aesthetics?